<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1014</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CD Grab</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="hLayout" stretch="0,1">
      <item>
       <widget class="QWidget" name="wDevice" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QVBoxLayout" name="vlDevice">
           <item>
            <widget class="QGroupBox" name="gbDev">
             <property name="title">
              <string>Device Settings</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <layout class="QFormLayout" name="formLayout">
                <item row="2" column="0">
                 <widget class="QLabel" name="laDevice">
                  <property name="text">
                   <string>Device</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QComboBox" name="cDevice">
                  <property name="currentText">
                   <string>/dev/cdrom</string>
                  </property>
                  <item>
                   <property name="text">
                    <string>/dev/cdrom</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>dev/dvd</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>/dev/sda</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="laDriveStatus">
                  <property name="text">
                   <string>Drive</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QLineEdit" name="leDrive">
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QLabel" name="laDiscStatus">
                  <property name="text">
                   <string>Disc</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="1">
                 <widget class="QLineEdit" name="leDisc">
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="gbActions">
             <property name="title">
              <string>Actions</string>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="2" column="2">
               <widget class="QPushButton" name="pbEject">
                <property name="text">
                 <string>Eject</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QPushButton" name="pbGrab">
                <property name="text">
                 <string>Grab</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QPushButton" name="pbEncode">
                <property name="text">
                 <string>Encode</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QPushButton" name="pbAbort">
                <property name="text">
                 <string>Abort</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QProgressBar" name="pbDevice">
             <property name="value">
              <number>0</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="wTags" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="vlTags">
           <item>
            <widget class="QGroupBox" name="gbAlbum">
             <property name="title">
              <string>Album</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <layout class="QFormLayout" name="flAlbum">
                <property name="bottomMargin">
                 <number>34</number>
                </property>
                <item row="2" column="0">
                 <widget class="QLabel" name="laUPC">
                  <property name="text">
                   <string>UPC</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLineEdit" name="leUPC"/>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="laTitle">
                  <property name="text">
                   <string>Title</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLineEdit" name="leTitle"/>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="laArtist">
                  <property name="text">
                   <string>Artist</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLineEdit" name="leArtist"/>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QTableView" name="tvTracks">
             <property name="verticalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOn</enum>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1014</width>
     <height>38</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
